### ISSUE TITLE
"Text Input" element contents in GUI does not match object's internal .value

### ISSUE THREAD

**Issue by evan-shelley** [zauberzeug/nicegui#1123]
### Description

When implementing a system that can be used to save & load the value of a number of GUI elements to an external file, I encountered an issue where the Text Input element in GUI does not match the object's internal .value after using .set_value more than once.

The code below can be used to replicate the issue. Entering text, saving the state of the input_box, changing the contents, loading the state, changing the contents, then loading it again will result in this bug. The logging prints can be used to see that the .value of the object does get correctly set, but this does not reflect in the GUI. Instead, the Text Input contents will remain unchanged.

```python
from nicegui import ui
import logging

global input_state
input_state = {"state":""}

def save_input_state(input_box):
    logging.info("Saving input box contents: {}".format(input_box.value))
    input_state["state"] = input_box.value

def load_input_state(input_box):
    logging.info("Loading input box contents from global: {}".format(input_state["state"]))
    input_box.set_value(input_state["state"])
    logging.info("Loaded input box contents: {}".format(input_box.value))

input_box = ui.input("Enter text here")
ui.button("Save State", on_click=lambda: save_input_state(input_box))
ui.button("Load State", on_click=lambda: load_input_state(input_box))

format = "%(asctime)s: %(message)s"
logging.basicConfig(format=format, level=logging.INFO, datefmt="%H:%M:%S")

ui.run(reload=False, native=True)

```

**Comment by BLucky-gh** [comment]
it worked for me with `ui.input`, but I'm getting the same behavior with `ui.textarea`, the value gets set, but doesn't get reflected in the GUI

**Comment by falkoschindler** [comment]
@evan-shelley Thanks for reporting this issue!
Can you, please, check your NiceGUI version? We fixed a similar bug in [version 1.2.23](https://github.com/zauberzeug/nicegui/releases/tag/v1.2.23)

@BLucky-gh As an inherited class `ui.textarea` should behave exactly like `ui.input`. Can you check your NiceGUI version and, if the problem persists, please post a code example reproducing the issue? Thanks!

**Comment by BLucky-gh** [comment]
> @BLucky-gh As an inherited class `ui.textarea` should behave exactly like `ui.input`. Can you check your NiceGUI version and, if the problem persists, please post a code example reproducing the issue? Thanks!
my version is `1.2.24`, as for a code sample reproducing the issue, I'll try making one later since my code is pretty large for a reproduction example

**Comment by evan-shelley** [comment]
> @evan-shelley Thanks for reporting this issue! Can you, please, check your NiceGUI version? We fixed a similar bug in [version 1.2.23](https://github.com/zauberzeug/nicegui/releases/tag/v1.2.23)

Thanks for the reply! It seems I was running on version 1.2.22, so I'd just missed out on those changes. Now on version 1.2.24, I tried with both ui.input and ui.textarea and it works as expected!
